src/javascript/app/common/form_validation.js

```diff
@@ -346,13 +347,17 @@ const Validation = (() => {
             }
 
             if (!field.is_ok) {
-                message = options.message || ValidatorsMap.get(type).message;
+                let message_template;
+                field.$error.text(options.message || ValidatorsMap.get(type).message);
                 if (type === 'length') {
-                    message = template(message, [options.min === options.max ? options.min : `${options.min}-${options.max}`]);
+                    message_template = template(message, [options.min === options.max ? options.min : `${options.min}-${options.max}`]);
+                    field.$error.text(message_template);
```

Comment:
I see you're setting this in every condition below as well, could you call it once below the entire `if statement` instead, on line 362?

Reply:
Fixed